cmd/bundle/init.go

```diff
@@ -49,20 +52,42 @@ func newInitCommand() *cobra.Command {
 	cmd.Flags().StringVar(&outputDir, "output-dir", "", "Directory to write the initialized template to.")
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
-		templatePath := args[0]
 		ctx := cmd.Context()
+		var templatePath string
+		if len(args) > 0 {
+			templatePath = args[0]
+		} else {
+			return errors.New("please specify a template")
+
+			/* TODO: propose to use default-python (once template is ready)
+			var err error
+			if !cmdio.IsOutTTY(ctx) || !cmdio.IsInTTY(ctx) {
+				return errors.New("please specify a template")
+			}
+			templatePath, err = cmdio.Ask(ctx, "Template to use", "default-python")
+			if err != nil {
+				return err
+			}
+			*/
+		}
+
+		profile := root.GetProfile(cmd)
+		w, err := databricks.NewWorkspaceClient(&databricks.Config{Profile: profile})
+		if err != nil {
+			return err
+		}
 
 		if !isRepoUrl(templatePath) {
 			// skip downloading the repo because input arg is not a URL. We assume
 			// it's a path on the local file system in that case
-			return template.Materialize(ctx, configFile, templatePath, outputDir)
+			return template.Materialize(ctx, w, configFile, templatePath, outputDir)
```

Comment:
Can we instead use `context.WithValue(ctx, &workspaceClient, w)` here. Then we don't need to change materialise function and it's also in like how we set workspaceClient when we do auth for other commands in CLI